test/cypress/tests/modules/BlockEvents/Backspace.cy.ts

```diff
@@ -0,0 +1,363 @@
+import type EditorJS from '../../../../../types/index';
+
+/**
+ * Creates Editor instance with list of Paragraph blocks of passed texts
+ *
+ * @param textBlocks - list of texts for Paragraph blocks
+ */
+function createEditorWithTextBlocks(textBlocks: string[]): void {
+  cy.createEditor({
+    data: {
+      blocks: textBlocks.map((text) => ({
+        type: 'paragraph',
+        data: {
+          text,
+        },
+      })),
+    },
+  });
+}
+
+describe('Backspace keydown', function () {
+  it('should just delete chars (native behaviour) when some fragment is selected', function () {
+    createEditorWithTextBlocks([
+      'The first block',
+      'The second block',
+    ]);
+
+    cy.get('[data-cy=editorjs]')
+      .find('.ce-paragraph')
+      .last()
+      .click()
+      .selectText('The ')
+      .type('{backspace}');
+
+    cy.get('[data-cy=editorjs]')
+      .find('div.ce-block')
+      .last()
+      .should('have.text', 'second block');
+  });
+
+  it('should just delete chars (native behaviour) when Caret is not at the start of the Block', function () {
+    createEditorWithTextBlocks([
+      'The first block',
+      'The second block',
+    ]);
+
+    cy.get('[data-cy=editorjs]')
+      .find('.ce-paragraph')
+      .last()
+      .click() // caret will be at the end of the block
+      .type('{backspace}');
+
+    cy.get('[data-cy=editorjs]')
+      .find('div.ce-block')
+      .last()
+      .should('have.text', 'The second bloc'); // last char is removed
+  });
+
+  it('should navigate previous input when Caret is not at the first input', function () {
+    /**
+     * Mock of tool with several inputs
+     */
+    class ExampleOfToolWithSeveralInputs {
+      /**
+       * Render method mock
+       */
+      public render(): HTMLElement {
+        const container = document.createElement('div');
+        const input = document.createElement('div');
+        const input2 = document.createElement('div');
+
+        container.setAttribute('data-cy', 'quote-tool');
+
+        input.setAttribute('contenteditable', 'true');
+        input2.setAttribute('contenteditable', 'true');
+
+        container.append(input, input2);
+
+        return container;
+      }
+
+      /**
+       * Saving logic is not necessary for this test
+       */
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
FYI: you can add a comment to ignore a rule like this

```suggestion
      // eslint-disable-next-line @typescript-eslint/no-empty-function -- saving logic is not necessary for this test
```